extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmProcessor.java

```diff
@@ -1466,6 +1489,12 @@ private PreGeneratedProxies generateProxies(Set<String> managedClassAndPackageNa
                     CachedProxy proxy = proxyCache.cache.get(managedClassOrPackageName);
                     generatedProxyQueue.add(CompletableFuture.completedFuture(proxy));
                 } else {
+
+                    if (!managedClassesName.contains(managedClassOrPackageName)) {
+                        // we don't generate proxies for packages:
+                        continue;
+                    }
```

Comment:
You could simply iterate on `managedClassesName` instead of iterating on `managedClassAndPackageNames`?